specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/expressRouteCircuit.json

```diff
@@ -0,0 +1,2019 @@
+        }
+      ],
+      "description": "Peering in an ExpressRouteCircuit resource."
+    },
+    "ExpressRouteCircuitPeeringListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExpressRouteCircuitPeering"
+          },
+          "description": "The peerings in an express route circuit."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Response for ListPeering API service call retrieves all peerings that belong to an ExpressRouteCircuit."
+    },
+    "ExpressRoutePeeringType": {
+      "type": "string",
+      "description": "The PeeringType. Possible values are: 'AzurePublicPeering', 'AzurePrivatePeering', and 'MicrosoftPeering'.",
+      "enum": [
+        "AzurePublicPeering",
+        "AzurePrivatePeering",
+        "MicrosoftPeering"
+      ],
+      "x-ms-enum": {
+        "name": "ExpressRoutePeeringType",
+        "modelAsString": true
+      }
+    },
+    "ExpressRoutePeeringState": {
+      "type": "string",
+      "description": "The state of peering. Possible values are: 'Disabled' and 'Enabled'",
+      "enum": [
+        "Disabled",
+        "Enabled"
+      ],
+      "x-ms-enum": {
+        "name": "ExpressRoutePeeringState",
+        "modelAsString": true
+      }
+    },
+    "ExpressRouteCircuitConnectionPropertiesFormat": {
+      "properties": {
+        "expressRouteCircuitPeering": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "Reference to Express Route Circuit Private Peering Resource of the circuit initiating connection."
+        },
+        "peerExpressRouteCircuitPeering": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "Reference to Express Route Circuit Private Peering Resource of the peered circuit."
+        },
+        "addressPrefix": {
+          "type": "string",
+          "description": "/29 IP address space to carve out Customer addresses for tunnels."
+        },
+        "authorizationKey": {
+          "type": "string",
+          "description": "The authorization key."
+        },
+        "circuitConnectionStatus": {
+          "type": "string",
+          "description": "Express Route Circuit Connection State. Possible values are: 'Connected' and 'Disconnected'.",
+          "enum": [
+            "Connected",
+            "Connecting",
+            "Disconnected"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "circuitConnectionStatus",
+            "modelAsString": true
+          }
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Provisioning state of the circuit connection resource. Possible values are: 'Succeded', 'Updating', 'Deleting', and 'Failed'."
```

Comment:
Succeded -> Succeeded as long as the typo isn't in the enum

Reply:
SDK code seems to override this prop with the one inherited from SubResource
https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Network/Management.Network/Generated/Models/ExpressRouteCircuitConnection.cs#L52
I'll double check what the server returns